Privacy.md

```diff
@@ -0,0 +1,30 @@
+#Privacy 
+This is a plain English summary of all of the components within Piwik which may affect your privacy in some way. Please keep in mind that if you use third party Themes or Apps with Piwik, there may be additional things not listed here.
+##Privacy for Piwik admins and website owners
+As admins is the Piwik admin and any other user acessing Piwik.In this section, we document how you can configure your Piwik service to better protect your privacy.
+###Automatic update check
```

Comment:
Move ` Automatic update check` below ` Keep your user details private` as this section is less important.